cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,249 @@ Cask Data Application Platform Release Notes
+	- Introducing Distributed Rules Engine - for Business Analysts for effectively manage rules for data transformation and data policy.
+
+New Features
+------------
+
+Data Pipelines Enhancements
+---------------------------
+
+- :cask-issue:`CDAP-12033` - Added a new splitter transform plugin type that can send output to different ports
+
+- :cask-issue:`CDAP-12136` - Added a union splitter transform that will send records to different ports depending on which type in the union it is.
+
+- :cask-issue:`CDAP-12137` - Added a splitter transform that splits records based on whether the specified field is null.
+
+- :cask-issue:`CDAP-12154` - Added Kafka Alert Publisher to publish alerts from a cdap pipeline to a kafka topic.
+
+- :cask-issue:`CDAP-12155` - Added a plugin that publishes alerts to CDAP TMS
+
+- :cask-issue:`CDAP-12167` - plugin prepareRun and onFinish methods now run in a separate transaction per plugin so that pipelines with many plugins will not timeout.
+
+- :cask-issue:`CDAP-12191` - All pipeline plugins now have access to the pipeline namespace and name through their context object.
+
+- :cask-issue:`CDAP-9107` - Adds feature that allows undoing and redoing of actions in pipeline Studio.
+
+- :cask-issue:`CDAP-10619` - UI: Add upgrade step when importing pipeline
+
+- :cask-issue:`CDAP-12034` - Added a way for pipeline plugins to emit alerts, and a new AlertPublisher plugin type that publishes those alerts
+
+- :cask-issue:`CDAP-12057` - Makes pipeline nodes bigger to be able to show the version and metrics on the node.
+
+- :cask-issue:`CDAP-12077` - Revamps pipeline connections, to allow dropping a connection anywhere on the node, and allow selecting and deleting multiple connections using the Delete key.
+
+- :cask-issue:`CDAP-12108` - Batch data pipelines support condition plugin types which can control the flow of execution of the pipeline.
+
+- :cask-issue:`CDAP-12456` - Condition plugins in the pipeline have access to the stage statistics such as number of input records, number of output records, number of error records generated from the stages which executed prior to the condition node.
+
+- :cask-issue:`CDAP-12110` - Implemented Apache Commons JEXL based condition plugin which is available by default for the batch data pipelines.
+
+- :cask-issue:`CDAP-12111` - Added support for plugins of plugins. This allows the parent plugin to expose some APIs that its own plugins will implement and extend
+
+- :cask-issue:`CDAP-12114` - UI: Add ability to support custom label and custom icons for pipeline plugins
+
+- :cask-issue:`CDAP-12236` - Implements the design for Condition, a special type of pipeline plugin.
+
+- :cask-issue:`CDAP-10974` - BatchSource, BatchSink, BatchAggregator, BatchJoiner, and Transform plugins now have a way to get SettableArguments when preparing a run, which allows them to set arguments for the rest of the pipeline
+
+- :cask-issue:`CDAP-11618` - UI: Add Kafka Connection in Data Preparation
```

Comment:
Added the ability for users to connect Data Preparation to their existing data in Apache Kafka

Reply:
Also, this should move to the next section on Data Preparation?